src/app/public/src/modules/swagger/swagger.component.html

```diff
@@ -0,0 +1 @@
+<div id="swagger-ui"></div>
```

Comment:
Would it be possible to replace this with a class name, and keep convention with the other components? This will keep the feature modular, and not tie a component instance to a specific identifier.

```
<div class="stache-swagger-ui" #swaggerWrapper></div>

// ...

swagger.SwaggerUIBundle({
  // dom_id: '#swagger-ui',
  domNode: this.swaggerWrapper
});
```

Reply:
It's not possible, since it's a requirement of the api, but I can change it to #stache-swagger-ui.

Reply:
@blackbaud-brandonhare : Did you see my inclusion of `domNode`? https://github.com/swagger-api/swagger-ui#parameters

Reply:
^ I'm setting `domNode` with a `ViewChild` reference...